FRAuth/FRAuth/Device/TelephonyCollector.swift

```diff
@@ -28,35 +29,53 @@ public class TelephonyCollector: DeviceCollector {
         var result: [String: Any] = [:]
         
         let networkInfo = CTTelephonyNetworkInfo()
-        var carrier: CTCarrier?
+        var carrier: CarrierInfo?
         
-        if #available(iOS 12.0, *) {
-            if let providers = networkInfo.serviceSubscriberCellularProviders, providers.keys.count > 0 {
-                for (_, thisCarrier) in providers{
-                    carrier = thisCarrier
-                    break
-                }
+        
+        if let providers = networkInfo.serviceSubscriberCellularProviders, providers.keys.count > 0 {
```

Comment:
i think we can simplify this to one line may be ?

 var carriers = providers.map { (carrierName: $0.value.carrierName , isoCountryCode: $0.value.isoCountryCode ) }